db/fixtures/miq_searches.yml

```diff
@@ -1,103 +1,103 @@
 ---
-- attributes: 
```

Comment:
My editor killed the trailing whitespace in this and other yml files - I'll put these back if it's considered too much noise.